sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/LocalDocumentStorage.java

```diff
@@ -114,6 +114,7 @@ private LocalDocumentStorage(DatabaseManager databaseManager) {
     }
 
     LocalDocumentStorage(Context context) {
+        // TODO: implement a multi-table listener
```

Comment:
Lets remove the TODO we might create user tables dynamically actually on sign in so maybe it does not make sense to have onCreate code for that.